code/game/gamemodes/wizard/artefact.dm

```diff
@@ -82,6 +82,55 @@
 	rend_desc = "Gently wafting with the sounds of endless laughter."
 	icon_state = "clownrender"
 
+
+//--- BECAUSE IT'S FUCKING HILARIOUS --//
+/obj/item/weapon/veilrender/uncharged
+	name = "wicked blade"
+	desc = "A replica of a notorious eldritch artifact of immense power. Legend tells that those who capture two powerful souls with it will weild the weapon of legends."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "render"
+	item_state = "render"
+	force = 5 //shitty wizard build quality
+	throwforce = 0
+	w_class = 3
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	charges = 1
+	spawn_type = /obj/singularity/narsie/wizard
+	spawn_amt = 1
+	activate_descriptor = "the blood geometer's prison"
+	rend_desc = "You hear the sound of laughing slaughter demons."
+	spawn_fast = 1
+	var/captured_ascendants = 0
+
+/obj/item/weapon/veilrender/uncharged/attack_self(mob/user)
+	if(charges > 0 && captured_ascendants==2)
+		new /obj/effect/rend(get_turf(user), spawn_type, spawn_amt, rend_desc, spawn_fast)
+		charges--
+		captured_ascendants = 0
+		user << "<span class='boldannounce'>You combine the awesome power of the three ascendants bend the bridge of reality, beckoning forth the blood geometer.</span>"

```

Comment:
I know. If the wizard succeeds and eats the two ascendants, I want to evoke a sense of terror in the last one with the arrival of a real god. The kind of terror you can only achieve by stripping away the comfortable safety that a player is given when they play an overpowered role.